pyproject.toml

```diff
@@ -24,8 +24,9 @@ classifiers = [
 ]
 dependencies = [
     "typer>=0.7.0",
-    "ghga-event-schemas==0.13.4",
-    "hexkit[akafka]>=0.10.0",
+    "ghga-event-schemas>=1.0.0",
+    "ghga-service-commons>=1.0.0",
```

Comment:
```suggestion
    "ghga-event-schemas>=3.0.0,<4",
    "ghga-service-commons>=1.2.0",
```

Reply:
Apparently I'm living in the past where everything is still 1.0.0